dev-haskell/hlint/hlint-3.2.3.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.6.9999
+
+COMMIT="6c6593e0ec0fe45640879946a181f706c2a38972"
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+inherit haskell-cabal
+
+DESCRIPTION="Source code suggestions"
+HOMEPAGE="https://github.com/ndmitchell/hlint#readme"
+SRC_URI=" https://github.com/ndmitchell/hlint/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="+gpl hsyaml +threaded test"
+RESTRICT="!test? ( test )"
+
+RDEPEND=">=dev-haskell/aeson-1.1.2.0:=[profile?]
+	>=dev-haskell/ansi-terminal-0.8.1:=[profile?]
+	>=dev-haskell/cmdargs-0.10:=[profile?]
+	>=dev-haskell/cpphs-1.20.1:=[profile?]
+	>=dev-haskell/data-default-0.3:=[profile?]
+	>=dev-haskell/extra-1.7.3:=[profile?]
+	dev-haskell/file-embed:=[profile?]
+	>=dev-haskell/filepattern-0.1.1:=[profile?]
+	=dev-haskell/ghc-lib-parser-8.10*:=[profile?]
+	=dev-haskell/ghc-lib-parser-ex-8.10.0*:=[profile?]
+	>=dev-haskell/refact-0.3:=[profile?]
+	dev-haskell/text:=[profile?]
+	>=dev-haskell/uniplate-1.5:=[profile?]
+	dev-haskell/unordered-containers:=[profile?]
+	dev-haskell/utf8-string:=[profile?]
+	dev-haskell/vector:=[profile?]
+	gpl? ( >=dev-haskell/hscolour-1.21:=[profile?] )
+	hsyaml? ( >=dev-haskell/hsyaml-0.2:=[profile?]
+			>=dev-haskell/hsyaml-aeson-0.2:=[profile?] )
+	!hsyaml? ( >=dev-haskell/yaml-0.5.0:=[profile?] )
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-1.18.1.3
+	test? ( dev-haskell/apply-refact )
+"
+
+S="${WORKDIR}/hlint-${COMMIT}"
+
+src_configure() {
+	haskell-cabal_src_configure \
+		--flags=ghc-lib \
+		$(cabal_flag gpl gpl) \
+		$(cabal_flag hsyaml hsyaml) \
+		$(cabal_flag threaded threaded)
+}
+
+src_test() {
+	hlint --test || die
```

Comment:
@trofi the default does not work, due to a lack of test "stanza". I need to fix this though, so that it calls the executable in `${S}`, just like I do in the hoogle

Reply:
I updated `src_test` in order to reflect the same idiom used in #1098